code/datums/components/onwear_mood.dm

```diff
@@ -0,0 +1,59 @@
+/// Add to clothing to give the wearer a mood buff and a unique examine text
+/datum/component/onwear_mood
+	/// the event the wearer experiences
+	var/datum/mood_event/saved_event
```

Comment:
Rename to `saved_event_type` to make it obvious this is a typepath and not an instance